codecov.gemspec

```
@@ -8,12 +8,27 @@ Gem::Specification.new do |s|
   s.summary               = 'Hosted code coverage'
   s.description           = 'Hosted code coverage Ruby reporter.'
   s.email                 = ['hello@codecov.io']
-  s.files                 = ['lib/codecov.rb', 'lib/codecov/version.rb']
-  s.homepage              = 'https://github.com/codecov/codecov-ruby'
+  s.files                 = Dir[
+    'lib/**/*.rb', 'README.md', 'LICENSE', 'CHANGELOG.md'
+  ]
   s.license               = 'MIT'
+  s.version               = ::Codecov::VERSION
+
+  github_uri = 'https://github.com/codecov/codecov-ruby'
+
+  s.metadata = {
+    'source_code_uri' => github_uri,
+    'documentation_uri' =>
+      "http://www.rubydoc.info/gems/#{s.name}/#{s.version}",
+    'bug_tracker_uri' => "#{github_uri}/issues",
+    'changelog_uri' => "#{github_uri}/blob/v#{s.version}/CHANGELOG.md"
+  }
+
+  s.metadata['homepage_uri'] = github_uri
+  s.homepage = s.metadata['homepage_uri']
```

```suggestion
  s.metadata = {
    'bug_tracker_uri' => "#{github_uri}/issues",
    'changelog_uri' => "#{github_uri}/blob/v#{s.version}/CHANGELOG.md",
    'homepage_uri' => github_uri,
    'documentation_uri' =>
      "http://www.rubydoc.info/gems/#{s.name}/#{s.version}",
    'source_code_uri' => github_uri
  }
  s.homepage = github_uri
```

alphabetized and removed some indirection. I don't expect us to have a separate homepage in the near-term future.

---

The point is in the thing that "homepage equals homepage" instead "this homepage equals GitHub URI and this homepage equals GitHub URI too", it's more logical and prevents multiple changes.

I've changed it a little to more beautiful way and sorted `metadata` keys alphabetically.